test/step-function.test.ts

```diff
@@ -883,7 +883,7 @@ test("put an event bus event", () => {
     {
       Parameters: { Entries: [{ EventBusName: bus.eventBusArn }] },
     },
-    "bus.putEvents"
+    "1__await bus.putEvents"
```

Comment:
What changed here

Reply:
More states, needed to be more specific with the state names.